manager/allocator/networkallocator/networkallocator.go

```diff
@@ -657,7 +658,12 @@ func (na *NetworkAllocator) resolveDriver(n *api.Network) (driverapi.Driver, str
 }
 
 func (na *NetworkAllocator) loadDriver(name string) error {
-	_, err := plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	var err error
+	if pg := na.drvRegistry.GetPluginGetter(); pg != nil {
+		_, err = pg.Get(name, driverapi.NetworkPluginEndpointType, plugingetter.Lookup)
+	} else {
+		_, err = plugins.Get(name, driverapi.NetworkPluginEndpointType)
+	}
```

Comment:
FYI, the older pattern was necessary when plugins were experimental and both stable and experimental daemons had to work with plugins. Today, this is not necessary.